tests/unit/test_prestoclient.py

```diff
@@ -177,3 +177,35 @@ def test_execute_query_get_port(self, sudo_mock, conn_mock, mock_presto_config):
         self.assertEqual(client.rows, [])
         self.assertEqual(client.next_uri, '')
         self.assertEqual(client.response_from_server, {})
+
+    def test_create_authorization_headers(self, mock_presto_config):
+        auth_headers = PrestoClient.create_auth_headers("Aladdin", "open sesame")
+        expected_auth_headers = {"Authorization": "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ=="}
+        self.assertEqual(auth_headers, expected_auth_headers)
+
+    @patch('prestoadmin.prestoclient.error')
```

Comment:
I'd argue that the negative tests are all redundant, based on my comments above.